suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml

```diff
@@ -189,38 +194,38 @@ tests:
           timeout: 300
       desc: Perform rgw tests
 
-  - test:
-      name: Upgrade ceph
-      desc: Upgrade cluster to latest version
-      module: test_cephadm_upgrade.py
-      polarion-id: CEPH-83573791
-      config:
-        command: start
-        service: upgrade
-        base_cmd_args:
-          verbose: true
-        benchmark:
-          type: rados                      # future-use
-          pool_per_client: true
-          pg_num: 128
-          duration: 10
-        verify_cluster_health: true
-      destroy-cluster: false
-      abort-on-fail: true
-
-  # Running basic rbd and rgw tests after upgrade
-  - test:
-      name: rbd-io
-      module: rbd_faster_exports.py
-      config:
-          io-total: 100M
-      desc: Perform export during read/write,resizing,flattening,lock operations
-
-  - test:
-      name: rgw sanity tests
-      module: sanity_rgw.py
-      config:
-          script-name: test_multitenant_user_access.py
-          config-file-name: test_multitenant_access.yaml
-          timeout: 300
-      desc: Perform rgw tests
+#  - test:
```

Comment:
Please remove the comments. As per polarion this test case is automated.

Reply:
Commented the following snippet, as once fi lands into cdn, we need to run these tests again. Rationale added at the top of suite file. The same upgrade test is run with deployment suite too.

Reply:
Commented as we can un-comment once the fix lands into CDN for the bug. Complete rationale added at the top of suite